src/vsphere_cpi/Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'builder',     '~>3.2.3'
 gem 'nokogiri',    '~>1.8.1'
 gem 'mono_logger', '~>1.1.0'
 gem 'oga', '2.3'
+gem 'typhoeus', '~> 1.0', '>= 1.0.1'
+gem 'json', '~> 2.1', '>= 2.1.0'
```

Comment:
Do we need this line? Ruby has JSON support builtin these days.

Reply:
This is for nsxt generated client